### DISCUSSION TITLE
togglecampaignteleport should make the map button always visible

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#13980] [Ideas and suggestions]
togglecampaignteleport is a command used to teleport between map nodes freely for testing purposes. It is useless if you're in the middle of a level or in the ending levels, where the map cannot be accessed.
For better usability, the command should let you open the map from anywhere, not just teleport between locations *if* you can access the map.

**Comment by Regalis11** [comment]
FYI you can reveal the entire map by enabling `debugdraw`. But I agree, this command should reveal it too.

**Reply by NotWendy** [reply]
No, I mean the ability to open the map menu in the first place. Normally, you can't do it from anywhere you want, you need to be in a specific part of the level.
